app/views/schedules/index.html.haml

```diff
@@ -10,11 +10,8 @@
         - if project_assignments.present?
           = render 'project', project: project, assignments: project_assignments, users: @schedule.users
 
-    - if @schedule.holiday_assignments.present?
-      = render 'project', project: Project.new(name: 'Holiday', project_state: 'leave'), assignments: @schedule.holiday_assignments, users: @schedule.users
-
-    - if @schedule.sick_assignments.present?
-      = render 'project', project: Project.new(name: 'Sick', project_state: 'leave'), assignments: @schedule.sick_assignments, users: @schedule.users
+    - @schedule.ooo_assignments.each_pair do |ooo_name, ooo_assignments|
```

Comment:
Worth using a different variable for the block parameter (you have two uses of `ooo_assignments` here) ?